base_images/Dockerfile.local

```diff
@@ -5,6 +5,7 @@ LABEL maintainer="Rizbi Hassan" \
 
 ENV NODE_VERSION="16.x"
 ENV TERRAFORM_VERSION "0.13.6"
+ENV PYTHON_VERSION "3.9.10"
```

Comment:
See notes on changes made to the bamboo Dockerfile and replicate here.

Reply:
updated the changes here as well